great_tables/gt.py

```diff
@@ -71,6 +71,20 @@
     create_heading_component_h,
     create_source_notes_component_h,
 )
+from great_tables._utils_render_latex import (
```

Comment:
Can you move these out of `gt.py` along with the definition `_render_as_latex()`? You could still import `_render_as_latex()` from somewhere and add it as a method.

But as it exists, this code is...

* importing `as_latex`, which...
* calls `_render_as_latex()`, that is defined in this file, which...
* calls the `create_*()` functions defined in `_utils_render_latex`

So the path is going...

* `_export.as_latex()` -> `gt.py` -> `_utils_render_latex()`

Ideally it seems like it should just be going...

* `_export.as_latex()` -> `_utils_render_latex()`

As then we just assign the function to GT.as_latex()

Reply:
Thanks! Completed during code pairing.